data/graphics.js

```diff
@@ -10009,11 +10058,60 @@ var BattleMoveAnims = {
 			}, 'decel', 'explode');
 		}
 	},
+	hydroshot: {
```

Comment:
If you're going to do something like this as a base point for moves, since it isn't a move itself, it needs to go in BattleOtherAnims